### ISSUE TITLE
example_keyword_spotting compile error on macOS

### ISSUE THREAD

**Issue by stephanschulz** [zkmkarlsruhe/ofxTensorFlow2#5]
I am on macOS 10.15.7 with OF 0.11.0 and Xcode 12.4 
I successfully compiled and ran all provided examples except `example_keyword_spotting`
Not sure why or how to fix it.
I don't have an immediate need for it but thought you might like to know.

`/Applications/of_v0.11.0_osx_release/addons/ofxTensorFlow2/src/ofxTensorFlow2Utils.h:119:14: Non-constant-expression cannot be narrowed from type 'size_t' (aka 'unsigned long') to 'long long' in initializer list`

<img width="1790" alt="Screen Shot 2021-02-19 at 11 09 39 AM" src="https://user-images.githubusercontent.com/1054816/108530095-3935e080-72a3-11eb-9b87-9d9863851f25.png">

Thanks,
Stephan.

**Comment by bytosaur** [comment]
sorry for late response... i am pretty sure that cast should not be `(size_t)` but `(shape_t)`. Could you please validate it for me?

**Comment by danomatika** [comment]
@bytosaur If that works, we should probably put out a 1.0.1 bug fix release and include a changeling. I will add one soon.

**Comment by stephanschulz** [comment]
yes that changed worked.
```
//	shape_ = {(size_t) srcVector.size()};
        shape_ = {(shape_t) srcVector.size()};
            
```

**Comment by danomatika** [comment]
This is now fixed and there is a new version tag: 1.0.1
